.travis.yml

```diff
@@ -37,9 +37,7 @@ matrix:
 script:
   - |
     if [[ $COMPOSER_CREATE = true ]]; then
-        composer create-project sulu/sulu-minimal ../test-create-project-dir --repository="{\"type\":\"path\",\"url\":\"./\"}" --stability=dev -n
-        cd ../test-create-project-dir
-        echo 'DATABASE_URL=mysql://root:@127.0.0.1:3306/sulu_test' >> .env.local
+        composer create-project sulu/sulu-minimal ../test-create-project-dir --repository="{\"type\":\"path\",\"url\":\"./\"}" --stability=dev -n && cd ../test-create-project-dir && echo 'DATABASE_URL=mysql://root:@127.0.0.1:3306/sulu_test' >> .env.local
```

Comment:
this change is needed else it will not exit correctly:
See: https://travis-ci.org/sulu/sulu-minimal/jobs/521738947